contents/founders/negotiate-software-better.md

```diff
@@ -0,0 +1,91 @@
+---
+date: 2024-06-04
+title: How software salespeople will try to screw you (and how to fight back)
+author:
+  - charles-cook
+featuredImage: https://res.cloudinary.com/dmukukwp6/image/upload/v1713344206/posthog.com/contents/blog/cracked_engineer_blog.jpg
+tags:
+  - Culture
+  - People
+crosspost:
+  - Blog      
+---
+
+At PostHog, we sell software. We also buy a lot of it - $320k in 2023 and that’s not including AWS or Stripe fees. This means we’ve spent a lot of time negotiating software contracts with the people we are buying from and selling to.
+
+I’m going to share some of the things we’ve learned that will hopefully help you negotiate a better deal with your software vendor, including PostHog!
+
+> Charles is the exec who oversees sales at PostHog, among other things. He is doing his best to try and save you money. 
+
+# Play the sales cycle
+
+Many software contract amounts are set by a salesperson trying to hit quota. If you know the end of their quarter, you can wait until that point to get a juicy discount.
+
+There are a couple of reasons why this happens:
```

Comment:
Two questions I still have:
- Why do they cram into this quarter vs wait for the next one?
- Why do they have so much flexibility on setting price?

Reply:
- Why do they cram into this quarter vs wait for the next one? - _I think I answered this (quarterly targets, commission), but maybe not explained well enough?_
- Why do they have so much flexibility on setting price? - _Hm not sure I have a good answer here because it's kinda random depending on what the company says they are allowed to do..._